configs/sst_kernel_rats-setup.yaml

```diff
@@ -4,23 +4,25 @@ data:
   name: rt-setup
   description: Package that creates realtime group and limits
   maintainer: sst_kernel_rats
-  packages:
-    - rt-setup
+  packages: []
   labels:
-    - eln
+  - eln
 
-  package-placeholders:
+  arch_packages:
```

Comment:
This probably won't do what you want. I assume you want to include the `rt-setup` as a placeholder only for x86_64, right? 

`arch_packages` pulls in packages from the repos the same way `packages` does. You need to add a `limit_arches` field under your placeholder, like this: https://github.com/minimization/content-resolver/blob/master/config_specs/workload.yaml#L102